app/views/npq_separation/admin/applications/reviews/_application_side_navigation.html.erb

```diff
@@ -1,16 +1,14 @@
 <% content_for :side_navigation do %>
-  <hr class="govuk-section-break govuk-section-break--m">
-
   <%=
     render NpqSeparation::SubNavigationComponent.new(
       request.path,
       heading: {
-        text: @application.user.full_name,
+        text: "#{@application.user.full_name}, #{@application.course.name}, #{@application.created_at.to_date.to_fs(:govuk)}",
```

Comment:
This looks very long for something being shown in a narrow side nav?

Reply:
Looking at the version it is replacing - by concatentating all the information together on a single row which will wrap in an inconsistent fashion I think we make this harder to read - not easier

Since the name is just duplicating the h1 title to the side of it - can it be something like the Date followed by a line break, followed by the Course name on the line below (possibly in a slightly smaller font to reduce the likelyhood of text wrapping)